ring_tx.h

```diff
@@ -30,12 +30,24 @@ static inline void kernel_may_pull_from_tx(struct tpacket2_hdr *hdr)
 
 static inline int pull_and_flush_tx_ring(int sock)
 {
-	return sendto(sock, NULL, 0, MSG_DONTWAIT, NULL, 0);
+	int ret = -1;
```

Comment:
There should be no need to initialize `ret` here, just declare it.

```suggestion
	int ret;
```